### PR TITLE
python: added class UnitSystem.

### PR DESCRIPTION
**Pull request by stefoss23**
In python, deck has a memberfunction that returns the UnitSystem.

### CODE REVIEW

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by stefoss23** [comment]
jenkins build this please

File: python/cxx/unit_system.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include <opm/parser/eclipse/Units/UnitSystem.hpp>
+
+#include "export.hpp"
+
+
+void python::common::export_UnitSystem(py::module& module)
+{
+    py::class_<UnitSystem>(module, "UnitSystem")
+        .def_property_readonly( "type", &UnitSystem::getName );
```
Comment (joakim-hove) [inline python/cxx/unit_system.cpp]: I think this property should be called `name`

**Comment by stefoss23** [comment]
jenkins build this please

File: python/cxx/unit_system.cpp (hunk 1 above)
Comment (stefoss23) [reply python/cxx/unit_system.cpp]: please note the jenkins failure.

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by stefoss23** [comment]
jenkins build this please

**Pull request merged**